ui/src/app/pages/profile/profile-page.spec.tsx

```diff
@@ -49,7 +49,7 @@ describe('ProfilePageComponent', () => {
     mockUpdateProfile = jest.spyOn(profileApi, 'updateProfile');
 
     // mocking because we don't have access to the angular service
-    reload.mockImplementation(async () => {
+    reload.mockImplementation(async() => {
```

Comment:
nit: This spacing surprises me. Should I be doing this? I kind of like it, but could go either way.

Reply:
The spacing is handled by the linter - but our TSLint rules are a bit all over the place in the repo. As these keep coming up, I think I am going to prioritize removing TSLint. I have some stories in the TD backlog

Reply:
Will the transition to ESLint resolve this?

Reply:
I think so. I went through the TSLint rules a while back and found that a lot of the rules we specify in some files are not getting applied at all, and we are instead using a different set of rules. The TSLint setup is a bit obfuscated right now.

TSLint is also deprecated in favor of ESLint and the TSLint team is working on getting all of the TS rules working in in ESLint. With ESLint, the rules specified in `.eslintrc.js` are definitely the rules that get applied, and get applied consistently. 

Also, linting the entire codebase and individual files is much faster with ESLint.